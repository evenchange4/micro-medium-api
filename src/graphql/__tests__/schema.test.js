const { addMockFunctionsToSchema } = require('graphql-tools');
const { graphql } = require('graphql');
const schema = require('../schema');

addMockFunctionsToSchema({ schema });

it('should return mocked post data', async () => {
  const query = `
  query QueryPost {
    posts(username: "username") { title }
  }
  `;

  expect(await graphql(schema, query)).toMatchSnapshot();
});

it('should return mocked user data', async () => {
  const query = `
  query QueryUser {
    user(username: "username") { bio }
  }
  `;

  expect(await graphql(schema, query)).toMatchSnapshot();
});
