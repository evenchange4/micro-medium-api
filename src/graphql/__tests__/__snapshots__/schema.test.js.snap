// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should return mocked post data 1`] = `
Object {
  "data": Object {
    "posts": Array [
      Object {
        "title": "Hello World",
      },
      Object {
        "title": "Hello World",
      },
    ],
  },
}
`;

exports[`should return mocked user data 1`] = `
Object {
  "data": Object {
    "user": Object {
      "bio": "Hello World",
    },
  },
}
`;
